cloudflare/resource_cloudflare_api_token_test.go

```diff
@@ -42,6 +42,29 @@ func TestAccAPIToken(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareAPITokenMultiplePermissionsGroups(t *testing.T) {
+	// Temporarily unset CLOUDFLARE_API_TOKEN if it is set as the API token
+	// endpoint does not yet support the API tokens without an explicit scope.
+	if os.Getenv("CLOUDFLARE_API_TOKEN") != "" {
+		defer func(apiToken string) {
+			os.Setenv("CLOUDFLARE_API_TOKEN", apiToken)
+		}(os.Getenv("CLOUDFLARE_API_TOKEN"))
+		os.Setenv("CLOUDFLARE_API_TOKEN", "")
+	}
+
+	rnd := generateRandomResourceName()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudflareAPITokenMultiplePermissionsGroups(rnd),
+			},
```

Comment:
for this to check the behaviour we're changing, we need another step to run a `diff` on the existing configuration too. we should just be able to duplicate the existing config and swap the values around. (no need to assert `ExpectEmptyPlan: true` as that is the default)

```suggestion
			{
				Config: testAccCloudflareAPITokenMultiplePermissionsGroups(rnd),
			},
			{
				Config: testAccCloudflareAPITokenMultiplePermissionsGroupsReordered(rnd),
			},
```

Reply:
fixed in c851cf3f84384bc0fe381235e23360828ffe15fd